xwiki-platform-core/xwiki-platform-icon/xwiki-platform-icon-fontawesome/src/main/resources/IconThemes/FontAwesome.xml

```diff
@@ -101,6 +101,7 @@ eject = eject
 step-forward = step-forward
 step-backward = step-backward
 fast-forward = fast-forward
+fast-backward = fast-backward
```

Comment:
Everything related to icon sets will be moved to a dedicated commit once the PR is in a ready to merge state.